test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/Output/DesignTime/Inject.cs

```diff
@@ -34,8 +34,12 @@ public class _TestFiles_Input_Inject_cshtml : global::Microsoft.AspNetCore.Mvc.R
         {
         }
         #pragma warning restore 1998
-        [global::Microsoft.AspNetCore.Mvc.Razor.Internal.RazorInjectAttribute]
-        public MyApp MyPropertyName { get; private set; }
+#line 2 "/TestFiles/Input/Inject.cshtml"
```

Comment:
Curious as to what this would do to the editor. Aka there'll be a visible line mapping for the type/property name but not for the get/setter which is what the debugger would try to highlight.